ruby/game_project/guessing_game.rb

```diff
@@ -1,31 +1,52 @@
 class GuessingGame
   
 	def initialize(word)
+	 	@attempts = 0
 		@word = word.downcase
 		@display_word = "_" * word.length
 	end
   
-  	# Take a guess in the form of a letter and check to see if it is in the
-  	# target word, update the word pattern to include the missing letter
+	# Take a guess in the form of a letter and check to see if it is in the
+	# target word, update the word pattern to include the missing letter
 	def guess_the_letter(g_letter)
+	 	@attempts += 1
```

Comment:
Is there a way to not add to the attempts if the user guesses the same letter more than once?